### PR TITLE
Improved test

### PR DESCRIPTION
**Pull request by axypas-phrasee**
For the time being just removed AntD.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,10 @@ The application requires refactoring and completion. Please complete the functio
 
 Feel free to use any technology available in the React environment to achieve the results.
 
+## Design
+The login page should look like, but do not make it pixel perfect.
```
Comment (nikos-poultsidis) [inline README.md:17]:
```suggestion
The login page should look like the following image, but doesn't have to be pixel perfect.
```

**Review by nikos-poultsidis — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -33,24 +40,18 @@ Feel free to use any technology available in the React environment to achieve th
         "username": "test.user@phrasee.co",
         "password": "testpassword"
     }
-    A successful login will have a response with a 200 HTTP status code
-    An unsuccessful login will have a response with a 401 HTTP status code
+   
 3.  After a successful response has been received, use it to render a page listing hospital patients following the design attached in 'digital-hospital-global-patients.jpg'.
 4.  This page should not be available to users who are 'unauthenticated'. Those users should be presented with the login page instead.
-5.  Please use Redux to manage the state of the app and place any HTTP requests in action dispatcher.
+5.  Favor the use Redux to manage the some of the state of the app.
```
Comment (russellphrasee) [inline README.md:46]: This is a bit weird grammatically, can we change it to "Whilst not a strict requirement, it is preferable that you use Redux to manage at least some of the state of the app." please?

**Review by russellphrasee — APPROVED** [review]
I left a comment about the readme but generally this looks good @axypas-phrasee

File: README.md (hunk 2 above)
Comment (axypas-phrasee) [reply README.md]: Thanks, done

**Pull request merged**